packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -121,16 +132,103 @@ const SideNavFooterContainer = styled("div", {
   },
 }));
 
+const getHasScrollableContent = (scrollableContainer: HTMLElement) =>
+  scrollableContainer.scrollHeight > scrollableContainer.clientHeight;
+
 const SideNav = ({
   navHeaderText,
   isCollapsible,
   onCollapse,
   onExpand,
   sideNavItems,
+  expandedWidth = "300px",
```

Comment:
I don't like hardcoding this magic number. We should move it out as a const.

Typically, these are in our design tokens, but this one obviously doesn't make sense to put there. Still, it's a static const value that we may need to reference in multiple places.